### PR TITLE
FA-91 loginCompose Module

### PR DESCRIPTION
**Pull request by brosscode**
FA-91 loginCompose, with a composeCore module

### CODE REVIEW

File: composeCore/src/androidTest/java/com/brandon/composecore/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.brandon.composecore
```
Comment (ForzenAcademy) [inline composeCore/src/androidTest/java/com/brandon/composecore/ExampleInstrumentedTest.kt]: Delete this

Comment (ForzenAcademy) [reply composeCore/src/androidTest/java/com/brandon/composecore/ExampleInstrumentedTest.kt]: Did you rebase? I will ask

Comment (brosscode) [reply composeCore/src/androidTest/java/com/brandon/composecore/ExampleInstrumentedTest.kt]: Removed again with another rebase

File: app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt [hunk 2]
```diff
@@ -6,6 +6,7 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.brandon.createaccount.core.usecase.CreateUserUseCase
+import com.brandon.logincompose.viewmodel.LoginViewModel
```
Comment (ForzenAcademy) [inline app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt]: Unused import?

File: loginCompose/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/drawable-v24/ic_launcher_foreground.xml]: Why is this moved to login compose? It's a launcher icon

File: loginCompose/src/main/res/drawable/ic_launcher_background.xml [hunk 4]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/drawable/ic_launcher_background.xml]: Same question

File: loginCompose/src/main/res/drawable/ic_launcher_background.xml [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+        android:fillColor="#00000000"
+        android:pathData="M69,19L69,89"
+        android:strokeWidth="0.8"
+        android:strokeColor="#33FFFFFF" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M79,19L79,89"
+        android:strokeWidth="0.8"
+        android:strokeColor="#33FFFFFF" />
+</vector>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/drawable/ic_launcher_background.xml]: For your ttfs that were moved, you should make a UI core that will take stuff like that. Those will be shared between compose and legacy ui

File: loginCompose/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: Launcher icons should stay in the launching module

File: loginCompose/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/values/colors.xml]: UI core

File: loginCompose/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<resources>
+    <string name="app_name">ForzenBook</string>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/values/strings.xml]: UI core

File: loginCompose/src/main/res/values/themes.xml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (ForzenAcademy) [inline loginCompose/src/main/res/values/themes.xml]: This is an xml theme that doesn't make sense to be in a feature module

File: app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt (hunk 2 above)
Comment (brosscode) [reply app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt]: Going to investigate this, I saw it in one of the files and it wasn't greyed out, meaning it got use somewhere, likely for a log constant if I had to guess

Will remove it since I shouldn't be pulling in another viewmodel in there anyways

**Review by ForzenAcademy — APPROVED** [review]

File: app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt (hunk 2 above)
Comment (brosscode) [reply app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt]: Removed this import, it was bringing in a log tag from the other viewmodel. Added a log tag for this viewmodel in the companion object

File: loginCompose/src/main/res/drawable-v24/ic_launcher_foreground.xml (hunk 3 above)
Comment (brosscode) [reply loginCompose/src/main/res/drawable-v24/ic_launcher_foreground.xml]: removed it

File: loginCompose/src/main/res/drawable/ic_launcher_background.xml (hunk 4 above)
Comment (brosscode) [reply loginCompose/src/main/res/drawable/ic_launcher_background.xml]: removed it, was unsure if they belonged in each module or not

File: loginCompose/src/main/res/drawable/ic_launcher_background.xml (hunk 5 above)
Comment (brosscode) [reply loginCompose/src/main/res/drawable/ic_launcher_background.xml]: removed this, now it lives in uiCore

File: loginCompose/src/main/res/values/colors.xml (hunk 7 above)
Comment (brosscode) [reply loginCompose/src/main/res/values/colors.xml]: removed from loginCompose

File: loginCompose/src/main/res/values/strings.xml (hunk 8 above)
Comment (brosscode) [reply loginCompose/src/main/res/values/strings.xml]: moved to uiCore

File: loginCompose/src/main/res/values/themes.xml (hunk 9 above)
Comment (brosscode) [reply loginCompose/src/main/res/values/themes.xml]: removed xml theme

File: loginCompose/src/main/res/mipmap-anydpi-v26/ic_launcher.xml (hunk 6 above)
Comment (brosscode) [reply loginCompose/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: removed
